examples/lb/members/base-computes.tf

```diff
@@ -0,0 +1,17 @@
+resource "vkcs_compute_instance" "compute" {
+  count           = 2
+  name            = "compute-instance-${count.index}"
+  flavor_id       = data.vkcs_compute_flavor.basic.id
+  security_groups = ["default"]
```

Comment:
you need a group w/ 8080 port access

Reply:
and ping, if you use it for health checing

Reply:
> you need a group w/ 8080 port access

added policies for 8080 8443 and icmp